question19/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="./css/style.css" rel="stylesheet" />
+    <script type="module" src="./js/newsTopics.js" defer></script>
+    <script type="module" src="./js/slideshow.js" defer></script>
```

Comment:
IMO 
> The defer attribute has no effect on [module scripts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) — they defer by default.

[ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attributes)

Reply:
Thank you for your review! 
I did not know it and I learned a lot!! It gave me a chance to look into it again.Thank you for pointing out and I removed 'defer' from script tag.
fixed commit : https://github.com/csakyo/homework/pull/33/commits/7d169cd028155953bf3115baa901e3e9da742b60